packages/gatsby-theme-docs/src/layouts/internals/layout-page-header-side.js

```diff
@@ -4,13 +4,17 @@ import { designSystem } from '@commercetools-docs/ui-kit';
 const LayoutPageHeaderSide = styled.aside`
   grid-area: page-header-side;
   display: none;
+  padding-left: ${designSystem.dimensions.spacings.m};
+  margin: ${designSystem.dimensions.spacings.m} 0
+    ${designSystem.dimensions.spacings.xl} 0;
+  border-left: 1px solid ${designSystem.colors.light.borderPrimary};
 
   @media screen and (${designSystem.dimensions.viewports.largeTablet}) {
     display: block;
     width: ${designSystem.dimensions.widths.pageNavigation};
   }
   @media screen and (${designSystem.dimensions.viewports.laptop}) {
-    width: ${designSystem.dimensions.widths.pageNavigationSmall};
+    width: ${designSystem.dimensions.widths.pageHeaderSideSmall};
```

Comment:
Why the changes here? 🤔

Reply:
<img width="520" alt="Bildschirmfoto 2020-11-24 um 01 04 49" src="https://user-images.githubusercontent.com/31767369/100029208-0f2e6880-2df1-11eb-8af3-bde2abab6f4d.png">

I noticed with this perticular screen size that the width of the layout is too wide for the banner...as you can see you can't see the end of the border or half of the external icon. So I thought it's better to change the layout width so that this can't occur with other components with full width that could be rendered in this layout.
What do you think? 🤔

Reply:
We're definitely not changing the Layout just for a Banner with limited lifetime. 

For me the Banner should behave like normal HTML layout, i.e. if a line is too short the text wraps.  If the business owners don't like that they need to come up with a differnt text.  

By the way: "user research program" is not a given name, so please don't capitalize it.  (or, if it's chose to be a given name, then capitalize it completely).

Reply:
Agree, the layout must not change. The link text should just wrap.